.eslintrc.js

```diff
@@ -35,6 +32,7 @@ module.exports = {
     'import/prefer-default-export': 'off',
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'warn',
+    'jsx-a11y/no-noninteractive-element-to-interactive-role': 'off',
```

Comment:
Add this rule instead of turning it off. It is actually useful.

```
This is a linting rule provided by the eslint-plugin-jsx-a11y package that checks if an interactive role attribute is applied to a non-interactive element in JSX code.

Interactive elements are elements that can be interacted with by users, such as buttons or links, and they should have an appropriate role attribute set to convey their purpose to assistive technologies used by users with disabilities. However, non-interactive elements, such as div or span, should not have an interactive role attribute, as it can cause confusion for users and accessibility issues.
```